benchmark/moe/launcher_round2.py

```
@@ -0,0 +1,209 @@
+
+def main():
+    mesh = build_mesh(ep_size=EP_SIZE, tp_size=1)
+    qconfig = QuantizationConfig(moe_weight_dtype=WEIGHT_DTYPE, moe_activation_dtype=None)
+
+    all_results = {}
+    total = len(CONFIGS) * len(TOKEN_COUNTS)
+    run_idx = 0
+
+    if proc == 0:
+        print(
+            f"\nRound 2 Tuning: {len(CONFIGS)} configs × {len(TOKEN_COUNTS)} tokens = {total} runs"
```

Ruff (0.15.7)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)